src/checker.rs

```diff
@@ -78,9 +73,8 @@ fn dns_error(err: &reqwest::Error) -> Option<String> {
 }
 
 impl Checker {
-    pub fn new(config: CheckerConfig) -> Self {
+    pub fn new(_config: CheckerConfig) -> Self {
```

Comment:
Do we still need to pass the config in here now?

Reply:
just left it for now, maybe I'll just remove it all together after this PR